signals/main.py

```diff
@@ -24,9 +25,19 @@ def run_main(schema, generator_name, data_models, core_data, project_name, api_u
     except SignalsError as e:
         print(str(e))
     else:
+        clear_generated_code_files()
+
         progress('Finished generating your files!')
 
 
+def clear_generated_code_files():
+    if os.path.isdir('./signals_code'):
```

Comment:
Once this is moved to the generator class then you can just use the BUILD_DIR variable in case we ever change or rename it